Encountify/Views/MapPage.xaml.cs

```diff
@@ -79,13 +80,20 @@ protected override void OnDisappearing()
             base.OnDisappearing();
         }
 
-        static public void LoadMarker(Map map, Marker marker, Color color)
+        static public async void LoadMarker(Map map, Marker marker, Color color)
         {
+            Geocoder geoCoder = new Geocoder();
+            Position position = new Position(marker.Latitude, marker.Longitude);
+
+            IEnumerable<string> possibleAddresses = await geoCoder.GetAddressesForPositionAsync(position);
+            string address = possibleAddresses.FirstOrDefault();
```

Comment:
I tested line 89 in Siberia, somewhere in Antlantic ocean and in forest. In always of these cases function return some address, though I couldn't find that it is always the case, so it may need some null checking?

Reply:
Fair point. Looks like Location(0, 0) returns nothing so I'll test it out and write a check for this, thanks!